colibre/config.yml

```diff
@@ -20,6 +20,66 @@ description_template: description.html
 
 # Location and description of additional figures
 scripts:
+  - filename: scripts/cosmic_log_SNIa_rate_CSFH_based.py
+    title: 'Cosmic SNIa rate'
+    caption: SNIa rate history calculated from the SFR.txt file produced by SWIFT.
+    section: SN Rate (cosmic - CSFH based)
+    output_file: log_SNIa_rate_history_based_on_CSFH.png 
+  - filename: scripts/cosmic_SNIa_rate_CSFH_based.py
+    title: 'Cosmic SNIa rate'
+    caption: SNIa rate history calculated from the SFR.txt file produced by SWIFT.
+    section: SN Rate (cosmic - CSFH based)
+    output_file: SNIa_rate_history_based_on_CSFH.png 
+  - filename: scripts/cosmic_log_CC_SN_rate_CSFH_based.py
+    title: 'Cosmic CC SN rate'
+    caption: CC SN rate history calculated from the SFR.txt file produced by SWIFT.
+    section: SN Rate (cosmic - CSFH based)
+    output_file: log_CC_SN_rate_history_based_on_CSFH.png 
+  - filename: scripts/cosmic_CC_SN_rate_CSFH_based.py
+    title: 'Cosmic CC SN rate'
+    caption: CC SN rate history calculated from the SFR.txt file produced by SWIFT.
+    section: SN Rate (cosmic - CSFH based)
+    output_file: CC_SN_rate_history_based_on_CSFH.png 
+  - filename: scripts/cosmic_SNIa_over_CC_SN_rate_CSFH_based.py
+    title: 'Cosmic SNIa rate / CC SN rate'
+    caption: SNIa rate over CC SN rate history calculated from the SFR.txt file produced by SWIFT.
+    section: SN Rate (cosmic - CSFH based)
+    output_file: SNIa_over_CC_rate_history_based_on_CSFH.png 
+  - filename: scripts/cosmic_SNIa_rate.py
+    title: 'Cosmic SNIa rate'
+    caption: SNIa rate history plotted directly from the SNIa.txt file produced by SWIFT.
+    section: SN Rate (cosmic - stochastic)
+    output_file: SNIa_rate_history.png
+  - filename: scripts/cosmic_log_SNIa_rate.py
+    title: 'Cosmic SNIa rate'
+    caption: SNIa rate history plotted directly from the SNIa.txt file produced by SWIFT.
```

Comment:
You have plots that show different things (e.g. log vs. linear SNIa rate). But you use the same caption and title for both plots, as if it was the same plot. Please be more specific and reflect in the caption and in the title what is different between the plots. E.g. if the plot is in long, then I would write `log` both in the caption and in the title.

The same comment applies also to the cosmic CC SN rate. And also SNIa / CC SN both CSFH based and not CSFH based.